Utils.xcodeproj/project.pbxproj

```diff
@@ -7,14 +7,15 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		B80AA66F1EFAD8F200FC46A9 /* ImageFetcher.swift in Sources */ = {isa = PBXBuildFile; fileRef = B80AA66E1EFAD8F200FC46A9 /* ImageFetcher.swift */; };
 		B83FC66F1EF05FB300BF1173 /* Utils.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B83FC6651EF05FB300BF1173 /* Utils.framework */; };
 		B83FC6741EF05FB300BF1173 /* UtilsTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = B83FC6731EF05FB300BF1173 /* UtilsTests.swift */; };
 		B83FC6761EF05FB300BF1173 /* Utils.h in Headers */ = {isa = PBXBuildFile; fileRef = B83FC6681EF05FB300BF1173 /* Utils.h */; settings = {ATTRIBUTES = (Public, ); }; };
-		B83FC6811EF0641E00BF1173 /* ImageFetcher.swift in Sources */ = {isa = PBXBuildFile; fileRef = B83FC6801EF0641E00BF1173 /* ImageFetcher.swift */; };
 		B83FC6861EF06D3A00BF1173 /* ReactiveCocoa.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B83FC6831EF06D3A00BF1173 /* ReactiveCocoa.framework */; };
 		B83FC6871EF06D3A00BF1173 /* ReactiveSwift.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B83FC6841EF06D3A00BF1173 /* ReactiveSwift.framework */; };
 		B83FC6881EF06D3A00BF1173 /* Result.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B83FC6851EF06D3A00BF1173 /* Result.framework */; };
 		B83FC68B1EF071BB00BF1173 /* ImagePickerService.swift in Sources */ = {isa = PBXBuildFile; fileRef = B83FC68A1EF071BB00BF1173 /* ImagePickerService.swift */; };
+		B8402F371F01AB160027639D /* URLSessionExtension.swift in Sources */ = {isa = PBXBuildFile; fileRef = B8402F361F01AB160027639D /* URLSessionExtension.swift */; };
```

Comment:
Ideally, only the lines regarding `URLSessionExtension` must be present among the changes.